homeassistant/components/hive/climate.py

```diff
@@ -43,21 +45,11 @@ def __init__(self, hivesession, hivedevice):
         """Initialize the Climate device."""
         self.node_id = hivedevice["Hive_NodeID"]
         self.node_name = hivedevice["Hive_NodeName"]
-        self.device_type = hivedevice["HA_DeviceType"]
```

Comment:
We need to keep DeviceType to support the original format for the updatesource and unique_id

Reply:
please read the code, I change nothing on that handling

Reply:
@pvizeli  the line ```self.device_type = hivedevice["HA_DeviceType"] ``` looks to be missing from the new code, this is needed to support the correct `self.data_updatesource` value and format